contracts/core/extensions/CoreIssuanceOrder.sol

```diff
@@ -328,8 +328,13 @@ contract CoreIssuanceOrder is
         private
         view
     {
-        //Declare set interface variable
+        // Declare set interface variable
         ISetToken set = ISetToken(_order.setAddress);
+        address[] memory requiredComponents = _order.requiredComponents;
+        uint256[] memory requiredComponentAmounts = _order.requiredComponentAmounts;
+
+        // Make sure maker token addressesis non-null
+        require(_order.makerToken != address(0), "ORDER_MAKER_TOKEN_INVALID");
```

Comment:
Why is this necessary?

Reply:
You can create an issuance order with a null makerToken with a positive makerTokenAmount - which doesn't make sense.

But a non-null makerToken with a 0 amount makes sense (if we're trying to describe a charity-related use case).

Reply:
Is it a security issue? This seems more like an assertion on the setProtocol.js level. The IssuanceOrder cannot be completed if the maker token is null addressed

Reply:
True, I don't think its a security issue - since null address doesn't have transfer. But it saves users gas (by not attempting transfers) who are potentially calling with a null maker token.